.github/workflows/main.yml

```diff
@@ -24,16 +29,16 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        node-version: [18.x, 20.x]
+        node-version: [20.x, 22.x]
     steps:
     - uses: actions/checkout@v4
+      with:
+        persist-credentials: false
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v4
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install
     - name: Run test with Node.js ${{ matrix.node-version }}
       run: |
-        cd test
-        npm install
-        npm run test
+        npm test
```

Comment:
Could move this up to the `run:` line.

Reply:
Sure. Any meaningful reviews? If not...can you just approve commits when you only have "nice to have" advice? Fiddly feedback without an approval just delays things unnecessarily...